run.sh

```diff
@@ -225,8 +226,7 @@ check_themes() {
 
     theme_url=${theme_url:-$theme_name}
 
-    WP theme is-installed "$theme_name"
-    if [ $? -eq 0 ]; then
```

Comment:
This got flagged by shellcheck as a redundancy (in two place). That's why it was changed.